templates/liquid/Measure.liquid

```diff
@@ -75,7 +75,7 @@
             </tr>
         {% endif %}
 
-        {% if Measure.status.exists() %}
+        {% if Measure.status.exists() and Measure.status.value | downcase != 'active' %}
```

Comment:
Don't need the `| downcase` bit and I think it's not FHIR Liquid Profile supported even if we did need it